Source/Csla.AspNetCore/Blazor/ActiveCircuitHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ActiveCircuitHandler.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>Circuit handler indicating if code in server-side Blazor</summary>
+//-----------------------------------------------------------------------
+#if NET5_0_OR_GREATER
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components.Server.Circuits;
+
+namespace Csla.AspNetCore.Blazor
+{
+  /// <summary>
+  /// Circuit handler indicating if code in server-side Blazor.
+  /// </summary>
+  public class ActiveCircuitHandler : CircuitHandler
+  {
+    /// <summary>
+    /// Gets a value indicating whether the current scope
+    /// is running in server-side Blazor.
+    /// </summary>
+    public bool IsServerSideBlazor { get; private set; }
```

Comment:
Personally I might be tempted to actually explicitly set the property to false here. I know that the default is false, so the difference at the point of execution is nothing, but I tend it put it in to explicitly indicate that I intend for it to remain unset/be unmodified in certain circumstances, just for readability.